launch-script/src/env/evm.rs

```diff
@@ -96,7 +96,7 @@ fn check_neon_evm(
                 .into())
             }
         },
-        || unreachable!(),
+        || Err(StateError::MissingUpgradeAuthority(*neon_evm_program_id).into()),
```

Comment:
This case means that the user specifies an invalid account instead of neon_evm_program_id. It will be better to rename this error to `InvalidProgram`.

Reply:
Added InvalidProgram error. Thank you!